packages/shopify-codemod/test/fixtures/assert/move-literals-to-expected-argument/basic.output.js

```diff
@@ -0,0 +1,11 @@
+assert.equal(foo, 5, 'error message');
+assert.equal(bar, 'foo');
+assert.equal(this.foo.bar(), 'foo');
+assert.equal(!bar, 'foo');
+assert.equal(foo, null);
```

Comment:
Does this handle them being used in the proper order, with an error message at the end? `assert.equal(foo, bar, 'error message')` should stay the same, just want to make sure it does.